satellite/audit/reporter.go

```diff
@@ -131,6 +131,7 @@ func (reporter *Reporter) recordAuditFailStatus(ctx context.Context, failedAudit
 		}
 	}
 	if len(failed) > 0 {
+		reporter.log.Debug("Error recording Audit Failed Nodes ", zap.Strings("Node IDs", failed.Strings()))
```

Comment:
It might be worth changing these statements to `log.Error`

Reply:
Oh, actually, I see that the errors themselves are being logged as `Error`. Debug is okay

Reply:
Nevermind, debug is good

Reply:
There is a possibility log is nil. I would do a check before calling.

Reply:
updated by initializing the log if it is nil